### PR TITLE
Low-level spells cast by high-level players will receive smaller bonu…

### PR DESCRIPTION
**Pull request by zwisus**
…ses from +healing and +spell damage.

until Patch 2.0.1

Signed-off-by: Zwisus <zlt988@gmail.com>

### CODE REVIEW

**Comment by billy1arm** [comment]
@zwisus - Please be careful of submodules when submitting pr's

File: src/game/Object/Unit.cpp [hunk 1]
```diff
@@ -5494,7 +5494,8 @@ int32 Unit::SpellBonusWithCoeffs(Unit* pCaster, SpellEntry const* spellProto, in
     else 
         { coeff = CalculateDefaultCoefficient(spellProto, damagetype); }
 
-    float LvlPenalty = CalculateLevelPenalty(spellProto);
+    //float LvlPenalty = CalculateLevelPenalty(spellProto);//[-ZERO] not need. http://wowwiki.wikia.com/wiki/Patch_2.0.1
+    float LvlPenalty = 1.0f;
```
Comment (ryancheung) [inline src/game/Object/Unit.cpp]: @billy1arm You can image the DPS of using Rank 1 Fireball spell with 800+spell damage benefit, if there's no low level penalty of spell(spells learned below 20).

Comment (zwisus) [inline src/game/Object/Unit.cpp]: @ryancheung  PLZ check db spell_bonus_data.

SQL: DELETE FROM `spell_bonus_data` WHERE (`entry`='133')

Comment (ryancheung) [inline src/game/Object/Unit.cpp]: @zwisus OK, even without the 100% bonus set in db, the default coefficient of Rank 1 fireball is 1.5/3.5 = 42.85%, which means the spell can deal 400+ damage with 800+damage benefit, so this is also blizzlike?

**Comment by zwisus** [comment]
@ryancheung  400+danage，you need 400 / 42.85% ,spell effecs 933.
 blizzlike has  spell effecs 933??

File: src/game/Object/Unit.cpp (hunk 1 above)
Comment (zwisus) [inline src/game/Object/Unit.cpp]: @ryancheung 400+danage，you need 400 / 42.85% ,spell effecs 933.
blizzlike has spell effecs 933??

Comment (zwisus) [inline src/game/Object/Unit.cpp]: [IMG]http://thumb.dumparump.com/379/b8P28kH.jpg[/IMG][/URL]

Comment (zwisus) [inline src/game/Object/Unit.cpp]: @ryancheung 
Two picture Classic and TBC
https://github.com/zwisus/download

Comment (billy1arm) [inline src/game/Object/Unit.cpp]: @zwisus - Should I add this as a DB update ?
"SQL: DELETE FROM  spell_bonus_data  WHERE ( entry ='133')"

Comment (zwisus) [inline src/game/Object/Unit.cpp]: @billy1arm  pull request database.

Comment (zwisus) [inline src/game/Object/Unit.cpp]: @billy1arm  yes.

Comment (billy1arm) [inline src/game/Object/Unit.cpp]: @zwisus - Thank You

Added in:
https://github.com/mangoszero/database/commit/2b9b5c409ab005ae9803f1727b831b97c1d125d5

**Pull request merged**